src/partials/about.html

```diff
@@ -1,3 +1,62 @@
 <section id="about" class="section about-section">
-  <div class="container">About us</div>
+  <div class="container">
+    <div class="about-card-container">
+      <h2 class="about-header">About <span class="about-marker">Us</span></h2>
+      <p class="about-paragraph">
+        We are dedicated to providing engaging and effective 
+        English language courses in a supportive and stimulating learning environment
+      </p>
+    </div>
+    <div class="about-list-container">
+      <ul class="about-list">
+        <li class="about-list-item">
+          <div class="about-list-item-container">
+            <h3 class="about-list-header">01</h3>
+            <p class="about-list-paragraph">
+              Expert, innovative teaching staff
+            </p>
+          </div>
+        </li>
+        <li class="about-list-item">
+          <div class="about-list-item-container">
+            <h3 class="about-list-header">02</h3>
+            <p class="about-list-paragraph">
+              Wide range of course levels and options
+            </p>
+          </div>
+        </li>
+        <li class="about-list-item">
+          <div class="about-list-item-container">
+            <h3 class="about-list-header">03</h3>
+            <p class="about-list-paragraph">
+              Multicultural learning environment
+            </p>
+          </div>
+        </li>
+        <li class="about-list-item">
+          <div class="about-list-item-container">
+            <h3 class="about-list-header">04</h3>
+            <p class="about-list-paragraph">
+              Comprehensive language support services
+            </p>
+          </div>
+        </li>
+      </ul>
+      <div class="about-picture">
+        <picture>
```

Comment:
use just <img> with source set, not picture (you can look in "proposal to leave app" section